src/Components/Forms/support/Field.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react"
+import { Field as FormikField } from "formik"
+import styled from "styled-components"
+import { default as Input, InputProps } from "../../Input"
+
+export interface FormikInputProps extends InputProps {}
+
+/**
+ * An input wrapped in a Formik Field for rendering convenience.
+ */
+export const Field: React.SFC<FormikInputProps> = props => {
+  const { name, type, placeholder, block, autoFocus } = props
+  return (
+    <FormikField
+      name={name}
+      type={type}
+      placeholder={placeholder}
+      // prettier-ignore
+      // because restProps should not have a trailing comma (fixed in prettier 1.9)
+      render={({
+        field, // { name, value, onChange, onBlur }
+        form: { touched, errors, values }, // also values, setXXXX, handleXXXX, dirty, isValid, status, etc.
+        ...restProps
+      }) => (
+        <div>
+          <Input
```

Comment:
The Formik `<Field>` wraps an `input` or other component and passes the formik form + field props into it.